docs/source/cn/type_tensor.py

```diff
@@ -0,0 +1,60 @@
+import oneflow
+from docreset import reset_docstr
+
+
+reset_docstr(
+    oneflow.BoolTensor,
+    r"""
+    创建一个张量，其数据类型为布尔型，它的参数与 :func:`oneflow.Tensor` 相同。
+    """,
+)
+
+reset_docstr(
+    oneflow.ByteTensor,
+    r"""
+    创建一个张量，其数据类型为 uint8，它的参数与 :func:`oneflow.Tensor` 相同。
+    """,
+)
+
+reset_docstr(
+    oneflow.CharTensor,
+    r"""
+    创建一个张量，其数据类型为 int8，它的参数与 :func:`oneflow.Tensor` 相同。
```

Comment:
```suggestion
    创建一个张量，其数据类型为 oneflow.int8，它的参数与 :func:`oneflow.Tensor` 相同。
```

这样是不是更准确，怕用户有误解，也就是说，用 Tensor 显示的真实的 oneflow 类型。

```
>>> a = oneflow.CharTensor()
>>> a
tensor([], dtype=oneflow.int8)
```

其它几个也一起改了吧。